### ISSUE TITLE
Build out a strategy for Go representation at non-go conferences

### ISSUE THREAD

**Issue by cassandraoid** [golang/cowg#7]
We could start with a list of conferences that have solid X language track (new, modern, emerging - doesn't matter) as well as open source confs. Could build out the list and open up to community to 'submit their talks' and attempt to get expert speakers at said conferences.

**Comment by spf13** [comment]
There's two sides to this. 1. track Go evangelism at Non-Go conferences. 2. Encourage / Empower more evangelism at Non-Go conferences.

**Comment by dlsniper** [comment]
We currently have no other language present at the Go conferences, would we want to have a Java track at GopherCon?

**Comment by nathany** [comment]
@dlsniper It may not be about taking Go to a Java conference. More like having Go represented at FOSDEM, OSCON, etc. -- which it already is too some extent :-)

**Comment by spf13** [comment]
Yes Nathan is correct. The intent is general conferences. Not language
specific ones.

**Comment by danicat** [comment]
Here in Brazil we have Qcon in São Paulo and The Developers Conference (TDC) in São Paulo, Florianópolis and Porto Alegre. TDC already has a Go track and I'm one of the organizers in the Porto Alegre edition that will happen in November.

My experience approving a Go track at TDC was very good, I had no problems doing so, but it is still hard to find speakers here. I've managed to build a pretty decent speaker line up with some very interesting talks, but I can't help but to feel that I was lucky as there weren't so many submissions.

I think the biggest challenge creating a language track is to prove that your community has a strong speaker line up. Even if there aren't many, you can't compromise on quality.
